docker/android-slave-s2i/contrib/bin/scl_enable

```diff
@@ -0,0 +1,3 @@
+# This will make scl collection binaries work out of box.
```

Comment:
Same as above. We shud not create another folder and use android-slave resources.
Let's avoid duplicates in the repo, which are hard to maintain and change.